sonar-db/src/main/resources/org/sonar/db/event/EventMapper.xml

```diff
@@ -47,9 +56,13 @@
       #{createdAt, jdbcType=BIGINT})
   </insert>
 
-  <delete id="delete">
+  <delete id="deleteById" parameterType="Long">
```

Comment:
As you've introduced UUID, I would recommend you to remove this deleteById method and use deleteByUuid (remember, the goal is to not use anymore ID on this table, it will make the removal of the column easier)